workflow/setup_expt.py

```diff
@@ -521,13 +524,32 @@ def validate_user_request(host, inputs):
             raise NotImplementedError(f"Supported resolutions on {machine} are:\n{', '.join(supp_res)}")
 
 
+def get_ocean_resolution(resdetatmos):
+    """
+    Method to determine the ocean resolution based on the atmosphere resolution
+    Limited options are going to be available
+    """
+    atmos_to_ocean_map = {
+        1152: 0.25, 768: 0.25, 384: 0.25,
+        192: 1.0,
+        96: 5.0, 48: 5.0}
```

Comment:
Should this table be added to RTD?

Reply:
The initial conditions page has the atmosphere and ocean resolutions we support but perhaps this info should be on a different page now that I look at it: https://global-workflow.readthedocs.io/en/latest/init.html